test/sequential/sequencerStaking.cancel.test.ts

```diff
@@ -300,3 +299,149 @@ it("GIVEN a sequencer, WHEN <in-correctly> canceling an update AND some pending
     await SequencerStaking.slashFineAmount(),
   );
 });
+
+it("GIVEN a sequencer, WHEN <in-correctly> canceling an update AND cancelator stakes new token THEN it will be slashed and returned to the sequencer pool", async () => {
+  await testUser1.refreshAmounts(AssetWallet.BEFORE);
+  const { reqIdCanceled, api } = await createAnUpdateAndCancelIt(
+    testUser1,
+    testUser2Address,
+    chain,
+  );
+  //await waitForNBlocks(disputePeriodLength);
+  await Rolldown.waitForReadRights(testUser2Address);
+  const txIndex = await Rolldown.lastProcessedRequestOnL2(chain);
+  //the cancellation is incorrectly
+  providingExtrinsic = await SequencerStaking.provideSequencerStaking(
+    (await SequencerStaking.minimalStakeAmount()).muln(2),
+    "Ethereum",
+  );
+  const cancelResolutionEvents = await Sudo.batchAsSudoFinalized(
+    Sudo.sudoAsWithAddressString(
+      testUser2Address,
+      new L2Update(api)
+        .withCancelResolution(txIndex, reqIdCanceled, false)
+        .on("Ethereum")
+        .build(),
+    ),
+    Sudo.sudoAs(testUser2, providingExtrinsic),
+  );
+  await testUser2.refreshAmounts(AssetWallet.BEFORE);
+  await waitSudoOperationSuccess(cancelResolutionEvents, "SudoAsDone");
+  await waitForNBlocks(disputePeriodLength);
+  await testUser2.refreshAmounts(AssetWallet.AFTER);
+  const sequencers = await SequencerStaking.activeSequencers();
+  const testUser2PenaltyValue = testUser2
+    .getAsset(GASP_ASSET_ID)
+    ?.amountBefore.reserved!.sub(
+      testUser2.getAsset(GASP_ASSET_ID)?.amountAfter.reserved!,
+    );
+  expect(sequencers.toHuman().Ethereum).toContain(testUser2Address);
+  expect(testUser2PenaltyValue).bnEqual(
+    await SequencerStaking.slashFineAmount(),
+  );
+});
+
+it("GIVEN a sequencer, WHEN <in-correctly> canceling an update AND some pending updates/cancels AND users stakes new token THEN they return sequencersRights", async () => {
```

Comment:
same as before, can not understand what is this test about

Reply:
Rename, pls check